heron/statemgrs/src/java/com/twitter/heron/statemgr/zookeeper/curator/CuratorStateManager.java

```diff
@@ -79,8 +79,8 @@ public void initialize(Config newConfig) {
 
       String newConnectionString = tunneledResults.first;
       if (newConnectionString.isEmpty()) {
-        throw new IllegalArgumentException("Cannot connect to tunnelHost: "
-            + tunnelConfig.getTunnelHost() + " Bad connectionString: " + connectionString);
+        throw new IllegalArgumentException("Cannot connect to tunnel host: "
+            + tunnelConfig.getTunnelHost());
```

Comment:
> For all of these exception messages, assume they get caught at the highest level, up in the python main and rendered to the user without any of the surrounding context that we have when reading the code.

very good suggestion! I followed this suggestion in #1593, but I'm sure there are parts I did not do perfectly. Feel free to point out.

Temporarily mark this one as resolved.